### DISCUSSION TITLE
Is there a way to get call stack from errors on try/catch in forge without using the -vvv

### DISCUSSION THREAD

**Discussion by maa105** [foundry-rs/foundry#6851] [General]
Hello we have a test that run in forge that is basically looping on external contact calls and checks if they pass. currently we just loop and try catch call and log if it passes or fails. but in the catch we do not have any info about the callstack that caused the error to be thrown. is there a cheat sheet where we can get the call stack inside the catch. we can get the reason but it is not very useful we would like to log the error stack as well if that is possible without failing the test completely

**Comment by DaniPopes** [comment]
I'm not sure I understand what you're asking for? You can catch the revert data with `catch(bytes memory revertData)` and log that
